code/datums/jobs.dm

```diff
@@ -1525,7 +1524,9 @@ ABSTRACT_TYPE(/datum/job/civilian)
 
 	New()
 		..()
-		src.access = get_all_accesses()
+		src.access = get_access("VIP")
+		src.access += get_access("Janitor")
+		return
```

Comment:
1. This feels like a janky way of giving them access. Could you just add a new section for `"Regional Director"` in `get_access()`? It's annoying but it's cleaner (one day someone will datumize access but for now we live with this shit)

2. Return isn't needed